resume.md

```diff
@@ -1,49 +1,47 @@
-* Launched PROJECT NAME to the Apple App Store resulting in X Downloads and Y Monthly Users
-* Beta Tested to X Number of users and collected Crash Data using analytics LIBRARY
-* ...
-
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Contributed to OPEN SOURCE LIBRARY, a popular module for FEATURE, by closing X tasks and merging Y Pull Requests.
-* Created Demo Tutorial series to demonstrate FEATURE for early adopters prior to production push.
-* Managed App Version release cycles to maintain proper software versioning for major and minor app increments.
-* ...
-
-## AWARDS/ACKNOWLEDGEMENTS
-* **COLLEGE Alumni Scholarship Recipient** - Alumni Scholarship Committee... *Month 20XX*
-* **1st Place Best Hack** - Hackathon Event... *Month 20XX*
-
-## Events/Conferences
-* **Cal Hacks 7.0 (remote)** - Berkeley, CA ... *Month 20XX*
-* **Cal Hacks 6.0** - Berkeley, CA ... *Month 20XX*
-* **WWDC 2019** - San Jose, CA ... *Month 20XX*
+**CS 128 Course Assistant**: *September 2020 - Present*
+* Course assistant for CS 128, a new core Computer Science class to be implemented Spring 2021
+* Developed engaging assignments in C++ for future students
+* Worked cohesively with other assistants to produce the course material
+
+**Sandia National Laboratories Space Blimp Challenge**: *November 2019*
+* Led team that developed new software that improves the management of experimental flight data 
+* Effectively utilized Java to parse through given data for maximum time efficiency
+
+**Alameda County and Engineering Fair**: *April 2018*
+* Presented a research study focused on Artificial Intelligence technologies and Machine Learning to industry professionals
+* Analyzed Artificial Intelligence services such as Amazon Rekognition Services, Google Cloud Vision Service, Microsoft’s Cognitive Services Vision and Face on their image correctness, accuracy, and comprehensiveness 
+
+## LEADERSHIP/ACTIVITIES
+**Illinois Formula Electric**: *August 2020 - Present*
+* Member of the Data Acquisition Analysis (DAQA) subsystem
+* Worked collaboratively with clubmates to enhance data visualization by using Python
+
+**Special Interest Group in Security (SIGPWNY)**: *August 2020 - Present*
+* Engaged in presentations focused around information security fundamentals
+* Competed in Capture The Flag cyber competitions
+
+**Special Interest Group Artificial Intelligence and Data Analytics (SIGAIDA)**: *October 2019 - Present*
+* Participated in comprehensive workshops focused around the applications of Machine Learning and Artificial Intelligence within the tech world
+* Gained experience with the use of PyTorch for neural network training 
+* Managing NLP Chatbot project written in Python
+
+**Special Interest Group Programing Languages (SIGPLAN)**: *October 2019 - Present*
+* Worked with students to reinforce knowledge of several high-level programming languages and algorithms 
+
+## SKILLS
```

Comment:
My two cents - add skills to the top this will help the reader understand what tools you are aware about.  Experience should emphasize *how* you use these tools

Reply:
would add any version control experience you have under your belt, git

^ consider putting this into a bucketed approach

* **Programming Languages**: Java, C, ...
* **Languages**: Mandarin Chinese (Fluent)
* **Enterprise Software (?)** Office, Drive, blah


```

Reply:
Can I pivot the Progamming Languages section to just Programming? And include git into this as I don't think I have enough to fill a separate Version Control section